aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -119,7 +119,7 @@ pub enum ScriptFunctionCall {
         coin_type: TypeTag,
         name: Vec<u8>,
         symbol: Vec<u8>,
-        decimals: u64,
+        decimals: u8,
```

Comment:
I committed a PR to fix this https://github.com/aptos-labs/aptos-core/pull/2975

Reply:
This is just generated I think. I'll revert any changes to `aptos-move` and see what happens.